bin/www

```diff
@@ -4,9 +4,10 @@
  * Module dependencies.
  */
 
-var app = require("../app");
```

Comment:
ma ata huligan

Reply:
switched to tick (') 'cause it's used everywhere